### DISCUSSION TITLE
Home tab same as YouTube Original

### DISCUSSION THREAD

**Discussion by SousukeAizen** [libre-tube/LibreTube#3609] [Ideas]
Make the home tab the same as YouTube Original and take the trends out of the home tab.

**Comment by hustlewithvikram** [comment]
Adding a home feed will be very good for those who want to watch videos suggested by libretube. Almost all people scroll through the YouTube home feed to find a good video to watch or they hit the trending tab for trending stuff to keep up with the people or just to be up to date.
